### DISCUSSION TITLE
Howto add comment to bru file

### DISCUSSION THREAD

**Discussion by wiquanAppD** [usebruno/bruno#595] [Q&A] [answered]
Howto add comments to bru file ?
It would be good to add documentation with the tests, etc.

```
## is this a comment ?
// is this a comment ?
/* is this a comment ? */
```

**Comment by helloanoop** [comment, marked as answer]
@wiquanAppD 

We have docs support for every request in `v0.24.0`. Would this solve your use case ?

![image](https://github.com/usebruno/bruno/assets/9350733/8e94dc7d-80ec-4794-9492-b24d747961d3)

**Reply by wiquanAppD** [reply]
yes!
